com.unity.shadergraph/Editor/Data/Nodes/Artistic/Adjustment/InvertColorsNode.cs

```diff
@@ -28,7 +28,7 @@ public override bool hasPreview
 
         string GetFunctionName()
         {
-            return $"Unity_InvertColors_{FindSlot<MaterialSlot>(OutputSlotId).concreteValueType.ToShaderString(concretePrecision)}";
+            return $"Unity_InvertColors_{FindSlot<MaterialSlot>(InputSlotId).concreteValueType.ToShaderString()}";
```

Comment:
Input and Output slots have the same concrete value type, but I thought Input makes more sense as it determines the output type.   We also keep the precision switchable here (using the $precision token) as we use $precision in the function code.